modules/cluster-logging-deploy-cli.adoc

```diff
@@ -58,7 +58,7 @@ spec:
   targetNamespaces:
   - openshift-logging <1>
 ----
-<1> You must specify the `openshift-logging` namespace for logging versions 5.7 and older. For logging 5.8 and later versions, you can use any namespace.
+<1> For logging versions 5.7 and older, the `openshift-logging` namespace must be specified. For logging versions 5.8 and later, the operator can be installed in the `openshift-logging` namespace, any other namespace, or in `All Namespaces`.
```

Comment:
That is not correct. For logging versions 5.8 and later, you can deploy openshift-logging instance in any namespace.

Reply:
apiVersion: operators.coreos.com/v1
kind: OperatorGroup
metadata:
  name: cluster-logging
  namespace: openshift-logging (1)
spec:
  targetNamespaces:
  - openshift-logging (2)
  
1  You must specify the `openshift-logging` namespace .
2  It is optional. You can leave it empty . that mean you can deploy openshift-logging crd in any namespace.

Reply:
Made the updates as discussed in [slack thread](https://redhat-internal.slack.com/archives/CB3HXM2QK/p1718710946706819).